nvim/apidef.go

```diff
@@ -441,10 +474,56 @@ func KeyMap(mode string) []*Mapping {
 	name(nvim_get_keymap)
 }
 
+// Commands gets a map of global (non-buffer-local) Ex commands.
+// Currently only user-commands are supported, not builtin Ex commands.
+//
+// opts is optional parameters. Currently only supports {"builtin":false}.
+func Commands(opts map[string]interface{}) map[string]interface{} {
+	name(nvim_get_commands)
+}
+
 func APIInfo() []interface{} {
 	name(nvim_get_api_info)
 }
 
+// SetClientInfo identify the client for nvim.
+//
+// Can be called more than once, but subsequent calls will remove earlier info, which should be resent if it is still valid.
+// (This could happen if a library first identifies the channel, and a plugin using that library later overrides that info)
+func SetClientInfo(name string, version map[string]interface{}, typ string, methods map[string]interface{}, attributes map[string]interface{}) {
+	name(nvim_set_client_info)
+}
+
+// Get information about a channel.
+func ChannelInfo(channel int) Channel {
+	name(nvim_get_chan_info)
+}
+
+// ListChannels get information about all open channels.
+func ListChannels() []Channel {
+	name(nvim_list_chans)
+}
+
+// ParseExpression parse a VimL expression.
+func ParseExpression(expr string, flags string, highlight bool) map[string]interface{} {
+	name(nvim_parse_expression)
+}
+
+// ListUIs gets a list of dictionaries representing attached UIs.
+func ListUIs() []interface{} {
```

Comment:
The Go idiomatic name is "UIs".   

Is it possible to define a struct type for a UI?  If a struct type is used, then result should be `[]*UI`.